lib/page_model.dart

```diff
@@ -142,7 +142,7 @@ class SinglePageModel extends PageModel {
   _processModel(YamlMap docMap) {
     super._processModel(docMap);
 
-    YamlMap viewMap = docMap['View'];
+    YamlMap viewMap = docMap['View'] ?? docMap;
```

Comment:
this doesn't seem right. It would mean that a page without the View is fine as well. Someone could ignore View and put options, menu, styles and footer and it would still try to execute it?

Reply:
I have made changes to that now. I have added 2 conditions in this. If (docMap.containsKey('View)) and if(docMap['View'] == null). If there is no view then it will throw error, otherwise it will check the value of view. If the value of view is null, then it will make an empty widget as a root widget, otherwise it will proceed as normal.